docs/index.asciidoc

```diff
@@ -80,6 +80,12 @@ Any ongoing modification of the dictionary file should be done using a
 copy/edit/rename or create/rename mechanism to avoid the refresh code from
 processing half-baked dictionary content.
 
+==== Compatibility with the Elastic Common Schema (ECS)
```

Comment:
```suggestion
[id="plugins-{type}s-{plugin}-ecs_metadata"]
==== Compatibility with the Elastic Common Schema (ECS)
```

Reply:
This is the anchor we've used on other ECS plugins.  If the word "metadata" is too off target for this application, we can pick another word as long as it doesn't create a duplicate anchor.